### PR TITLE
[Examples]: Add wasi-crypto example.

### PR DESCRIPTION
**Pull request by MediosZ**
Solve #1680 
Signed-off-by: Tricster <mediosrity@gmail.com>

### CODE REVIEW

**Review by hydai — CHANGES_REQUESTED** [review]
This PR should have a README to tell how to build, how to execute, and what's the expected output of this example.

**Comment by MediosZ** [comment]
Ok, I will try to add one.

File: examples/plugin/wasi-crypto/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+## Run the example
+
+We can run this example with `wasmedge` like
+
+```sh
+wasmedge target/wasm32-wasi/debug/wasi-crypto-example.wasm
+```
+
+This example should run successfully and print out the signatures.
```
Comment (hydai) [inline examples/plugin/wasi-crypto/README.md:49]: Is the output the same every time? If so, please post the execution output for users to check if the result is exactly matched.

**Review by hydai — CHANGES_REQUESTED** [review]
Because this example only showed some signature-related functions, I suggest that the name should change to wasi-crypto-signature for better meaning.

**Comment by MediosZ** [comment]
I change the name of this example to `wasi-crypto-signature`. This is ok cause I only have signature related examples.
I believe we can have a different repo for `wasi-crypto` rust bindings like `wasmedge_wasi_socket`, this will be useful for developers who want to use `wasi-crypto` in rust and WasmEdge.

**Review by hydai — APPROVED** [review]
LGTM

**Pull request merged**